site/class-1-bar/japgroevemaker/readme.md

```diff
@@ -0,0 +1 @@
+# Hello world
```

Comment:
This bar chart looks good (but it’s late), so you should also add the readme from the Readme?! assignment of class 2 to get points for it!